models.py

```diff
@@ -1,11 +1,24 @@
 from tortoise.models import Model
 from tortoise import fields
 
+class User(Model):
+	email = fields.charField(max_length=80)
```

Comment:
Does this actually work? I think the correct field is `CharField`, not `charField`.

Reply:
typo